File: Ohtap E Commerce Domain Models/Development/PHP/Objects/ProductAttributes/ProductAttributeBasic.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\Promotions;



use Ohtap\ECommerce\DomainModels\Objects\Attributes\IAttribute;
use Ohtap\ECommerce\DomainModels\Objects\Attributes\IAttributeData;
use Ohtap\ECommerce\DomainModels\Objects\Discounts\IDiscountData;

class ProductAttributeBasic implements IProductAttributeData
{

    public string $Description;
    public IAttributeData $Attribute;
    public string $Value;
    public int $State;
    public int $OrderNo;
    public string $Name;
    public string $ID;

}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Serializable Objects/ProductAttributes/ProductAttributeBasic.cs
using Ohtap.ECommerce.Domain.Models.Serializable.Objects.Attributes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ohtap.ECommerce.Domain.Models.Objects.Attributes;
using Ohtap.ECommerce.Domain.Models.Objects.ProductAttributes;

namespace Ohtap.ECommerce.Domain.Models.Serializable.Objects.ProductAttributes
{
    public class ProductAttributeBasic : IProductAttributeData
    {
        public int State { get; set; }
        public int Order { get; set; }
        public string ID { get; set; }
        public string Description { get; set; }
        public string Value { get; set; }
        public IAttributeData Attribute { get; set; }
    }
}
